docs/workflow/requirements/windows-requirements.md

```diff
@@ -34,7 +34,7 @@ Visual Studio 2019 installation process:
 
 A `.vsconfig` file is included in the root of the dotnet/runtime repository that includes all components needed to build the dotnet/runtime repository. You can [import `.vsconfig` in your Visual Studio installer](https://docs.microsoft.com/en-us/visualstudio/install/import-export-installation-configurations?view=vs-2019#import-a-configuration) to install all necessary components.
 
-The dotnet/runtime repository requires at least Visual Studio 2019 16.6.
+The dotnet/runtime repository requires at least Visual Studio 2019 16.10 (Preview).
```

Comment:
This document talks about prequisited for building the repo. Visual Studio preview should not be needed for building the repo. For example, the machines in CI have Visual Stuido 16.9.4 and the build works on them just fine.

The Visual Studio preview is typically only needed to develop the C# libraries with all bells and whistles.

Reply:
OK, should we require 16.9 (for net6.0 TFM).

Reply:
cc: @ViktorHofer

Reply:
Maybe this should be rephrased to "Building individual libraries in this repository requires at least Visual Studio 2019 16.10"?

Reply:
16.10 Preview is not required for building. It is only required for the VS IDE to work.

I would rephrase this as: "Visual Studio 2019 16.6 or later is required for building the repository. Visual Studio 2019 16.10 (Preview) is required to work with the libraries projects inside the Visual Studio IDE."